config/locales/pt-BR/views.yml

```diff
@@ -58,6 +58,12 @@ pt-BR:
       registered:
         one: "%{count} hora"
         other: "%{count} horas"
+  vacation_mailer:
+    notify_vacation_request:
+      info: "%{user} solicitou férias"
+      start_date: "Date de início: %{date}"
```

Comment:
data

Reply:
Good catch!

Reply:
Fixed that: https://github.com/Codeminer42/Punchclock/pull/294/commits/10966b5080c08ad710b846edb5b69e0ff641b538 ✅ 
Thanks!